apps/web/src/actions/sdk/runDocumentAction.ts

```diff
@@ -28,18 +25,17 @@ export async function runDocumentAction({
   commitUuid,
   parameters,
 }: RunDocumentActionProps) {
-  const sdk = await createSdk().then((r) => r.unwrap())
-  const stream = createStreamableValue<ChainEvent, Error>()
-  const response = sdk.runDocument({
-    params: {
-      projectId,
-      commitUuid,
-      documentPath,
-      parameters,
-      source: LogSources.Playground,
-    },
-    onMessage: (chainEvent) => {
-      stream.update(chainEvent)
+  const sdk = await createSdk(projectId).then((r) => r.unwrap())
```

Comment:
projectid can now be a constructor argument so you don't have to repeat it everywhere